.github/workflows/interface-unit-tests.yml

```diff
@@ -519,18 +519,10 @@ jobs:
          }}
       matrix:
         config:
-          - device: default.qubit.legacy
-            shots: None
           - device: default.qubit
             shots: None
           - device: default.qubit
             shots: 10000
-          - device: default.qubit.legacy
-            shots: 10000
-          # - device: default.qubit.tf
-          #   shots: None
-          - device: default.qubit.autograd
-            shots: None
             skip_interface: jax,tf,torch
```

Comment:
This `skip_interface` line should also be removed since now it's attached to the `default.qubit` case.